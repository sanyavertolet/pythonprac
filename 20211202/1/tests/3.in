class Squarator(metaclass=dump):
    def __init__(self, value):
        self.value = value
    
    def square(self):
        self.value = self.value * self.value
        return self

    def __str__(self):
        return str(self.value)

squarator = Squarator(2)
for i in range(10):
    print(squarator.square())
