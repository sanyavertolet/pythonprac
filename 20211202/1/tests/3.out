__init__: (2,), {}
square: (), {}
__str__: (), {}
4
square: (), {}
__str__: (), {}
16
square: (), {}
__str__: (), {}
256
square: (), {}
__str__: (), {}
65536
square: (), {}
__str__: (), {}
4294967296
square: (), {}
__str__: (), {}
18446744073709551616
square: (), {}
__str__: (), {}
340282366920938463463374607431768211456
square: (), {}
__str__: (), {}
115792089237316195423570985008687907853269984665640564039457584007913129639936
square: (), {}
__str__: (), {}
13407807929942597099574024998205846127479365820592393377723561443721764030073546976801874298166903427690031858186486050853753882811946569946433649006084096
square: (), {}
__str__: (), {}
179769313486231590772930519078902473361797697894230657273430081157732675805500963132708477322407536021120113879871393357658789768814416622492847430639474124377767893424865485276302219601246094119453082952085005768838150682342462881473913110540827237163350510684586298239947245938479716304835356329624224137216
